mmcv/utils/config.py

```diff
@@ -528,6 +528,15 @@ def __iter__(self):
     def __getstate__(self):
         return (self._cfg_dict, self._filename, self._text)
 
+    def __deepcopy__(self, memo):
+        other = self.__class__()
```

Comment:
```suggestion
        cls = self.__class__
        other = cls.__new__(cls)
```

Reply:
What is the purpose of this modification?
The current style is from the `__deepcopy__` function of `ConfigDict` (inherited from the `Dict` of `addict`)

Reply:
There is no need to initialize the object.